sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -36,4 +43,35 @@ export class InstitutionLocationAESTController extends BaseController {
       institutionId,
     );
   }
+
+  /**
+   * Controller method to retrieve institution location by id.
+   * @param institutionId
+   * @param locationId
+   * @returns institution location.
+   */
+  @Get(":locationId/institution/:institutionId")
+  async getInstitutionLocation(
+    @Param("institutionId") institutionId: number,
+    @Param("locationId") locationId: number,
+  ): Promise<InstitutionLocationFormAPIOutDTO> {
+    return this.locationControllerService.getInstitutionLocation(
+      institutionId,
+      locationId,
+    );
+  }
+
+  /**
+   * Update an institution location.
+   * @param locationId
+   * @param payload
+   * @returns InstitutionLocation.
+   */
+  @Patch(":locationId")
+  async update(
+    @Param("locationId") locationId: number,
```

Comment:
Generally our patch APIs are void. Is there a reason why we are returning here?

Reply:
We use the .save in the service, so it return the model itself, I have changed to InstitutionLocation